src/utils/ensureGithubFork.mjs

```diff
@@ -0,0 +1,27 @@
+function getRepo(api, repo) {
+  return new Promise((resolve, reject) => {
```

Comment:
I think this could be simplified if written as an async function.
```
try {
  await api.repos.get(...);
  console.log(' - ok, found fork');
  return;
} catch (e) {
  console.log(' - repo not found, forking');
}
 try {
    await api.repos.fork({...});
    console.log(' - forked repo');
    return;
} catch (e) {
  console.warn(' - error forking repo');
  throw e;
}
```

Reply:
Hah yeah totally. I forget that the node-github thing uses promises now, and i wrote them ;-)